draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -491,7 +492,7 @@ In {{example-json-minified}} the same content is shown with minified JSON (no
 whitespace, no blank lines) with ```'\'``` line wrapping per {{?RFC8792}}.
```

Comment:
We should remove the 8792 trailing note if we don't use it in the example

Reply:
Fixed, updated draft to use '\'